.gitignore

```diff
@@ -1,17 +1,9 @@
-*.pyc
-*.pyo
-*.gourmet-plugin
-*.desktop
-*~
-*#*
-.project
-.pydevproject
-.settings/
+# Byte-compiled / optimized / DLL files
+__pycache__/
+
+
+# Distribution / packaging
 *.egg-info/
-debian
-dist/
-build
```

Comment:
`build` is the output folder of `flatpak` builds, as with the wheels, it's built by the CI